feat: SQSCANGHA-6 add ability to pass certificate to action (#37)
SonarSource/sonarqube-scan-action: README.md

        LC_ALL: "ru_RU.UTF-8"
```

> If your sonarqube server users a self-signed certificate, you can pass a root certificate (in PEM format) to use for validation to the java certificate store:

Reviewer: Nitpicking:

```suggestion
If your SonarQube server users a self-signed certificate, you can pass a root certificate (in PEM format) to use for validation to the java certificate store:
```

> If your SonarQube server users a self-signed certificate, you can pass a root certificate (in PEM format) to use for validation to the java certificate store:

Reviewer: ```suggestion
If your SonarQube server uses a self-signed certificate, you can pass a root certificate (in PEM format) to the java certificate store:
```

```yaml
    - name: SonarQube Scan
      uses: sonarsource/sonarqube-scan-action@master
      env:
        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
        SONAR_HOST_URL: ${{ secrets.SONAR_HOST_URL }}

>         SONAR_ROOT_CERT: %{{ secrets.SONAR_ROOT_CERT }}

Reviewer: ```suggestion
        SONAR_ROOT_CERT: ${{ secrets.SONAR_ROOT_CERT }}
```

- `SONAR_TOKEN` – **Required** this is the token used to authenticate access to SonarQube. You can read more about security tokens [here](https://docs.sonarqube.org/latest/user-guide/user-token/). You can set the `SONAR_TOKEN` environment variable in the "Secrets" settings page of your repository, or you can add them at the level of your GitHub organization (recommended).
- `SONAR_HOST_URL` – **Required** this tells the scanner where SonarQube is hosted. You can set the `SONAR_HOST_URL` environment variable in the "Secrets" settings page of your repository, or you can add them at the level of your GitHub organization (recommended).

> - `SONAR_ROOT_CERT` – This is used to pass the scanner a custom root certificate (in PEM format), that should be used for validating the sonarqube server certificate. You can set the `SONAR_ROOT_CERT` environment variable in the "Secrets" settings page of your repository, or you can add them at the level of your GitHub organization (recommended).

Reviewer 2: ```suggestion
- `SONAR_ROOT_CERT` – Holds an additional root certificate (in PEM format) that is used to validate the SonarQube server certificate. You can set the `SONAR_ROOT_CERT` environment variable in the "Secrets" settings page of your repository, or you can add them at the level of your GitHub organization (recommended).
```
